packages/simple-hub/src/blockchain/asset-holder-watcher.ts

```diff
@@ -1,33 +1,33 @@
 import {ethers} from 'ethers';
-import {logger} from '../logger';
 import {ethAssetHolder as attachEthAssetHolder} from './asset-holder';
+import {fromEvent, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {bigNumberify, BigNumber} from 'ethers/utils';
 
-const log = logger();
+type RawAssetHolderEvent = [
+  string, //channelId
+  string, //amountDeposited
+  string, //destinationHoldings
+  ethers.Event //event
+];
 
-export enum AssetHolderWatcherEventType {
-  Deposited
-}
-
-export interface AssetHolderWatcherEvent {
+export interface AssetHolderEvent {
   channelId: string;
-  amountDeposited: string;
-  destinationHoldings: string;
+  amountDeposited: BigNumber;
+  destinationHoldings: BigNumber;
```

Comment:
Why switch to `BigNumber`?

Reply:
I find it easier to do arithmetic on `BigNumber`s. Not that we are doing much of that on the _simple-hub_ at the moment. Any objections?